README.md

```diff
@@ -46,7 +46,8 @@ submodule-cloning techniques:
   `git submodule update --init --remote`
 
 > NOTE: At any time during development you can use the submodule command to
-> refresh submodules: &nbsp;`git pull; git submodule update --init --remote`
+> refresh submodules:<br>
+> &nbsp;`git pull; git submodule update --init --remote`
```

Comment:
No need for the `&nbsp;`, right? Maybe use ``` instead?

Reply:
Yeah, good pt. I had just decided to leave it, but that's better.